package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "chrome-extension-webpack",
-  "version": "0.2.0",
+  "name": "your-turn-pr-review",
```

Comment:
modified manually, there is definitely a right way to do this

Reply:
This file is generated by `npm install`, so you could have run it to apply all changes for you